using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -1274,9 +1274,7 @@ FlutterACPIdentity.syncIdentifier("identifierType", "identifier", ACPMobileVisit
 
 ## syncIdentifiers
 
-The function of this API is the same as the `syncIdentifier` API. This API passes a list of identifiers, and each identifier contains an `identifier type` as the key and an `identifier` as the value. In each identifier pair, if the `identifier type` contains a null or an empty string, the identifier is ignored by the Identity extension.
```

Comment:
Just changed the order for syncIdentifiers and syncIdentifiers (overloaded) in this file.

Reply:
Let's change the description here as the current simply states it overrides another API. I think it would be appropriate to copy the description for the other `syncIdentifiers` API and additionally state the state is set to UNKNOWN. 


>The function of this API is the same as the `syncIdentifier` API. This API passes a list of identifiers, and each identifier contains an `identifier type` as the key and an `identifier` as the value. In each identifier pair, if the `identifier type` contains a null or an empty string, the identifier is ignored by the Identity extension.
For each identifier, the authentication state is set to the default value of `VisitorID.AuthenticationState.UNKNOWN`.